integrations/terraform-cloud/client.py

```diff
@@ -34,40 +36,26 @@ def __init__(self, terraform_base_url: str, auth_token: str) -> None:
         self.client = http_async_client
         self.client.headers.update(self.base_headers)
 
+        self.rate_limiter = AsyncLimiter(NUMBER_OF_REQUESTS, NUMBER_OF_SECONDS)
+
     async def send_api_request(
         self,
         endpoint: str,
         method: str = "GET",
         query_params: Optional[dict[str, Any]] = None,
         json_data: Optional[dict[str, Any]] = None,
     ) -> dict[str, Any]:
-        logger.info(f"Requesting Terraform Cloud data for endpoint: {endpoint}")
-        try:
```

Comment:
is there a reason why we no longer need to wrap the request in a try catch block?